frontend/resourceadm/components/NewResourceModal/NewResourceModal.test.tsx

```diff
@@ -9,32 +9,46 @@ import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
 import { ServicesContextProvider } from 'app-shared/contexts/ServicesContext';
 import { queriesMock } from 'app-shared/mocks/queriesMock';
 
-describe('NewResourceModal', () => {
-  const mockOnClose = jest.fn();
+const mockButtonText: string = 'Mock Button';
+const org = 'orgname';
+
+const mockOnClose = jest.fn();
+
+const defaultProps: NewResourceModalProps = {
+  onClose: mockOnClose,
+};
 
-  const defaultProps: NewResourceModalProps = {
-    isOpen: true,
-    onClose: mockOnClose,
-  };
+const mockedNavigate = jest.fn();
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedNavigate,
+  useParams: () => {
+    return { selectedContext: org };
+  },
+}));
+
+const user = userEvent.setup();
```

Comment:
This should be defined inside each of the it blocks as [recommended by the testing-library](https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent) 😄